packages/lib/shared/utils/text-format.ts

```diff
@@ -0,0 +1,3 @@
+export function formatTextListAsItems(items: string[]): string {
+  return items.reduce((acc: string, item: string) => acc + (acc ? '\n' : '') + '- ' + item, '')
```

Comment:
According to Groninge's screenshot, the alerts VS tootip items are displayed in reversed order.
```suggestion
  return items.reverse().reduce((acc: string, item: string) => acc + (acc ? '\n' : '') + '- ' + item, '')
```